debian/rules

```diff
@@ -1,114 +1,25 @@
 #!/usr/bin/make -f
-# -*- makefile -*-
-# Sample debian/rules that uses debhelper.
-# GNU copyright 1997 to 1999 by Joey Hess.
 
-# Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
-CFLAGS = -Wall -Wno-format -g
+export DEB_CFLAGS_MAINT_APPEND  = -Wall -Wno-format -Wno-format-security -g -DTAP_AND_HOLD
+export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
 
-ifneq (,$(findstring noopt,$(DEB_BUILD_OPTIONS)))
-	CFLAGS += -O0
-else
-	CFLAGS += -O2
-endif
-ifeq (,$(findstring nostrip,$(DEB_BUILD_OPTIONS)))
-	INSTALL_PROGRAM += -s
-endif
-ifneq (,$(findstring thumb,$(DEB_BUILD_OPTIONS)))
-        CFLAGS += -mthumb
-endif
 ifeq (,$(findstring nolauncher,$(DEB_BUILD_OPTIONS)))
 	CONFIGURE_OPTIONS += --enable-maemo-launcher
 	USE_MAEMO_LAUNCHER = true
 endif
 
-CFLAGS += -DTAP_AND_HOLD
-
-CXXFLAGS = $(CFLAGS)
-
-config.status:
-	if [ ! -x configure ]; then ./autogen.sh; fi
-	dh_testdir
-	# Add here commands to configure the package.
-	CFLAGS="$(CFLAGS)" CXXFLAGS="$(CXXFLAGS)" ./configure --prefix=/usr --sysconfdir=/etc --mandir=\$${prefix}/share/man --infodir=\$${prefix}/share/info --disable-static $(CONFIGURE_OPTIONS)
```

Comment:
Does this "--disable-static" change something?

Reply:
The compilation creates a few static libraries along with the shared objects. Creating both is standard in Debian packages, and unless there is a strong reason not to build the static libraries, I would leave it like this.

Reply:
I'd rather keep --disable-static if .a is just build but not installed in the -dev package (which I suspect it isn't). Building static libs just increases compile time of the package.

Reply:
It's installed in the -dev package. Reverted in 780bdf22fd425ba5acfd175baf70609250d7907c though. If the need arises, we can build static libraries again with a simple edit to debian/rules.